flexmeasures/data/models/forecasting/model_spec_factory.py

```diff
@@ -126,6 +123,7 @@ def create_initial_model_specs(  # noqa: C901
     custom_model_params: Optional[
         dict
     ] = None,  # overwrite model params, most useful for tests or experiments
+    time_series_class: Optional[type] = TimedBelief,
```

Comment:
My thought was to keep it around here, in case someone wants to use TBSeriesSpecs with a different time series class (and possibly with a different search method on that class, hence the existence of`search_fnc`).

Reply:
Okay I agree.